webview/src/plots/components/App.test.tsx

```diff
@@ -283,17 +283,68 @@ describe('App', () => {
     expect(loading).toHaveLength(3)
   })
 
-  it('should render the Add Plots and Add Experiments get started button when there are experiments which have plots that are all unselected', async () => {
```

Comment:
Tried my best to cover all the possibilities for the get started screen and to make the rather lengthy text clear 😅